docs/api/covidcast-signals/quidel.md

```diff
@@ -68,60 +68,66 @@ $$
 p = \frac{100 x}{n}
 $$
 
-We estimate p across 3 temporal-spatial aggregation schemes:
+We estimate p across 6 temporal-spatial aggregation schemes:
+- daily, at the county level;
 - daily, at the MSA (metropolitan statistical area) level;
 - daily, at the HRR (hospital referral region) level;
-- daily, at the state level.
+- daily, at the state level;
+- daily, at the HHS level;
+- daily, at the US national level.
 
-**MSA and HRR levels**: In a given MSA or HRR, suppose $$N$$ COVID tests are taken
-in a certain time period, $$X$$ is the number of tests taken with positive
-results. 
+#### Standard Error
 
-For raw signals:
-- if $$N \geq 50$$, we simply use:
+We assume the estimates for each time point follow a binomial distribution. The
+estimated standard error then is:
 
 $$
-p = \frac{100 X}{N}
+\text{se} = 100 \sqrt{ \frac{\frac{p}{100}(1- \frac{p}{100})}{N} } 
 $$
 
-For smoothed signals, before taking the temporal pooling average,
-- if $$N \geq 50$$, we also use:
+#### Smoothing
+
+We add two kinds of smoothing to the smoothed signals:
+
+##### Temporal Smoothing
+Smoothed estimates are formed by pooling data over time. That is, daily, for
+each location, we first pool all data available in that location over the last 7
+days, and we then recompute everything described in the two subsections above. 
+
+Pooling in this way makes estimates available in more geographic areas, as many areas 
+report very few tests per day, but have enough data to report when 7 days are considered.
+
+##### Geographical Smoothing
+
+**County, MSA and HRR levels**: In a given County, MSA or HRR, suppose $$N$$ COVID tests 
+are taken in a certain time period, $$X$$ is the number of tests taken with positive
+results. 
+
+
+For smoothed signals, after taking the temporal pooling,
+- if $$N \geq 50$$, we still use:
 $$
 p = \frac{100 X}{N}
 $$
-- if $$25 \leq N < 50$$, we lend $$50 - N$$ fake samples from its home state to shrink the
+- if $$25 \leq N < 50$$, we lend $$50 - N$$ fake samples from its parent state to shrink the
 estimate to the state's mean, which means:
 $$
 p = 100 \left( \frac{N}{50} \frac{X}{N} + \frac{50 - N}{50}  \frac{X_s}{N_s} \right) 
 $$
 where $$N_s, X_s$$ are the number of COVID tests and the number of COVID tests
-taken with positive results taken in its home state in the same time period.
+taken with positive results taken in its parent state in the same time period.
+A parent state is defined as the state with the largest proportion of the population 
+in this county/MSA/HRR.
 
-**State level**: the states with fewer than 50 tests are discarded. For the
-rest of the states with sufficient samples,
+Counties with sample sizes smaller than 50 are merged into megacounties for 
+the raw signals; counties with sample sizes smaller than 25 are merged into megacounties for
+the smoothed signals.
```

Comment:
I would provide either a short description of megacounties or a link to a description. "For example, the megacounty aggregating counties 20001, 20002, 20003, is 20000."

Reply:
Did check the @krivard, we have a general description of the megacounty [here](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_geography.html#small-sample-sizes-and-megacounties), so no more introduction here.